src/workloads/execution/MultiDeletesBatched_2MB.yml

```diff
@@ -0,0 +1,26 @@
+SchemaVersion: 2018-07-01
```

Comment:
The Datalake might be helpful here. https://wiki.corp.mongodb.com/display/DW There should be a lot of monitoring data to pick from. Note though, that it will be averages per collection or database (e.g. database size and number of documents).

Reply:
@ykangpark thanks for the patience. Would you be open to add another workload for 16kB-sized documents, both with and without the 7 secondary indexes?

I'm inclined to auto-run the 1kB tests as we already do, and introduce the 10MB tests, both including the secondary indexes variants. That would be 6 more auto-running tests introduced in this patch, for a total of 8. @dalyd does this sound reasonable to you? Also, pardon my ignorance: does make a test manual involve removing the `AutoRun` section in the workload? Could you point me to instructions how to run those tests manually?

Reply:
So 2 sizes (1 KB and 10 MB) vs 4 configs for 8 total. That sounds reasonable to me.